packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -0,0 +1,97 @@
+import SfOverlay from "../../atoms/SfOverlay/SfOverlay.vue";
+
+export default {
+  name: "SfModal",
+  props: {
+    /**
+     * Cross closing modal button
+     */
+    cross: {
+      type: Boolean,
+      default: true
+    },
+    /**
+     * Visibility of the modal
+     */
+    value: {
```

Comment:
```suggestion
    visible: {
```
I was talking about visible behavior to @filrak, and he said we shoud use props not v-model, then you can change this name to more user friendly. Of course if you use v-model you can too changes default names https://vuejs.org/v2/api/#model